### ISSUE TITLE
1.4.4 Resize text

### ISSUE THREAD

**Issue by svetlanaangelova** [samheather/iapt2group1#99]
- Login buttons are cut off. 
- Search description - <b> cannot resolve

@j6mes, can you please add a screenshot for this because I can't reproduce?
